charts/argo-cd/README.md

```diff
@@ -538,7 +538,7 @@ NAME: my-release
 | repoServer.env | list | `[]` | Environment variables to pass to repo server |
 | repoServer.envFrom | list | `[]` (See [values.yaml]) | envFrom to pass to repo server |
 | repoServer.extraArgs | list | `[]` | Additional command line arguments to pass to repo server |
-| repoServer.extraContainers | list | `[]` | Additional containers to be added to the repo server pod |
+| repoServer.extraContainers | string | `nil` | Additional containers to be added to the repo server pod |
```

Comment:
Why is this transitioning to a string from a list?

Reply:
Fixed - this was leftover in values.yaml as I was testing the sample.